packages/main/src/Button.ts

```diff
@@ -291,6 +296,9 @@ class Button extends UI5Element implements IFormElement, IButton {
 	@property({ type: Boolean })
 	_isTouch!: boolean;
 
+	@property({ type: Boolean })
```

Comment:
IMO we can add `noAttribute: true` here, as when we click on a button and then press Shift or Escape it applies a `_cancel-action` attribute, exposing an internal property intended for managing state, what do you think ?

Reply:
I think it is a good idea. Done.